### PR TITLE
feat(web): display in app survey in mobile view

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
AVO-318

## Description
This PR updates the styling of the in app survey card to display on mobile.

### CODE REVIEW

File: web/src/utils/styling.ts [hunk 1]
```diff
@@ -32,3 +32,11 @@ export function useBreakpoint(breakpoint: Breakpoint) {
...
   return useMediaQuery(queries[breakpoint]);
 }
+
+export function useIsMobile() {
+  return !useBreakpoint('sm');
+}
+
+export function useIsBiggerThanMobile() {
+  return useBreakpoint('sm');
+}
```
Comment (silkeholmebonnen) [inline web/src/utils/styling.ts:42]: I don't think this is used. Is it there for future use or should we delete it?

**Review by silkeholmebonnen — COMMENTED** [review]
Looks good! Just one comment:)

File: web/src/utils/styling.ts (hunk 1 above)
Comment (tonypls) [reply web/src/utils/styling.ts:42]: Good spotting, hoping to do some slack week work to use it!

**Review by silkeholmebonnen — APPROVED** [review]
LGTM ⭐

**Pull request merged**